### DISCUSSION TITLE
buddhism/eightfold/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#49] [General]
# buddhism/eightfold/

https://shanenull.com/buddhism/eightfold/

**Comment by shane0** [comment]
The four inverted views (Sanskrit: viparyasa, Pali: vipallasa) are described in the Pali cannon1 as the following:

Regarding impermanent phenomena as permanent

Regarding phenomena as having self when they have no self.

Regarding unpleasant (dukkha) phenomena as pleasant (sukkha)

Regarding unwholesome2 (asubha, literally, “un-lovely”3) phenomena as wholesome

**Comment by shane0** [comment]
Brilliant explanations 
https://youtu.be/UZNxbEKf6LA?si=ZydgHVPiCg5Fbemd
